Extensions/fonts/mcclim-clx-fonts.asd

```diff
@@ -0,0 +1,5 @@
+
+(defsystem #:mcclim-clx-fonts
+  :depends-on (#:mcclim-fonts/truetype #:mcclim-clx)
+  :components
+  ((:file "xrender-fonts")))
```

Comment:
Can we make it part of `mcclim-fonts.asd` file? The system name would be `mcclim-fonts/clx-truetype` and put below `mcclim-fonts/truetype`. Also `mcclim-clx/pretty` system must be modified to reflect this change (see `Backends/CLX/mcclim-clx.asd` - dependency should be adjusted to the new name).